var/spack/repos/builtin/packages/libgridxc/package.py

```diff
@@ -32,6 +31,10 @@ class Libgridxc(AutotoolsPackage):
     version("0.8.0", sha256="ff89b3302f850d1d9f651951e4ade20dfa4c71c809a2d86382c6797392064c9c")
     version("0.7.6", sha256="058b80f40c85997eea0eae3f15b7cc8105f817e59564106308b22f57a03b216b")
 
+    build_system(conditional("autotools", when="@1.1.0:"), "makefile", default="autotools")
```

Comment:
A package supporting multiple build systems should define `*Builder` classes explicitly. Can you add them?